security/src/test/java/com/twilio/security/storage/EncryptedPreferencesTest.kt

```diff
@@ -74,6 +74,25 @@ class EncryptedPreferencesTest {
     encryptedPreferences.put(key, value)
   }
 
+  @Test
+  fun testPut_notSavingInPreferences_shouldThrowError() {
+    val key = "key"
+    val value = "value"
+    val editor: Editor = mock()
+    whenever(preferences.edit()).thenReturn(editor)
+    whenever(serializer.toByteArray(value)).thenReturn(value.toByteArray())
+    whenever(secretKeyProvider.encrypt(any())).thenReturn(value.toByteArray())
+    whenever(editor.putString(eq(generateKeyDigest(key)), any())).thenReturn(editor)
+    whenever(editor.commit()).thenReturn(false)
+    exceptionRule.expect(StorageException::class.java)
+    exceptionRule.expectCause(
+        instanceOf(
+            IllegalStateException::class.java
+        )
```

Comment:
<table data-meta="generated_by_Linter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-android/blob/5b0eb61d7d2ddb2c184e8c32f649596ce888d592/security/src/test/java/com/twilio/security/storage/EncryptedPreferencesTest.kt#L91"></span>Unexpected indentation (8) (should be 6)</td>
    </tr>
  </tbody>
</table>